debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/RomClassWalker.java

```diff
@@ -631,31 +636,41 @@ private void allSlotsInConstantPoolDo() throws CorruptDataException
 				J9ROMFieldRefPointer ref = J9ROMFieldRefPointer.cast(cpEntry);
 				classWalkerCallback.addSlot(clazz, SlotType.J9_SRPNAS, ref.nameAndSignatureEA(), "cpFieldNAS");
 				classWalkerCallback.addSlot(clazz, SlotType.J9_U32, ref.classRefCPIndexEA(), "cpFieldClassRef");
-				
+
```

Comment:
looks like we are missing `J9CPTYPE_INTERFACE_STATIC_METHOD` and `J9CPTYPE_INTERFACE_INSTANCE_METHOD` here

Reply:
Fixed in the commit I've just added